packages/embed/src/utils/build-iframe.src.spec.ts

```diff
@@ -68,6 +69,7 @@ describe('build-iframe-src', () => {
           '&embed-opacity=50' +
           '&disable-tracking=true' +
           '&disable-auto-focus=true' +
+          '&__dangerous-disable-submissions=true' +
```

Comment:
a-ha! so this one is still useful ;)

Reply:
Yes and its also something we mention in our public docs, as I recently found out :)